src/js/components/DataTable/stories/typescript/OnSelect.tsx

```diff
@@ -25,16 +25,9 @@ const columns: ColumnConfig<RowType>[] = [
     header: 'Location',
     render: (datum) =>
       datum.location ? (
-        <Box
-          pad={{ horizontal: 'small', vertical: 'xsmall' }}
-          onClick={() => {}}
-          hoverIndicator
-          fill
-        >
-          <Text truncate weight="bold">
-            {datum.location}
-          </Text>
-        </Box>
+        <Text truncate weight="bold">
```

Comment:
Any reason we changed this?

Reply:
took this out so we can test how it is without having a row clickable since we wouldn't recommend doing this I can put back in